### ISSUE TITLE
wp-config Connection Settings for integrated users

### ISSUE THREAD

**Issue by raymond-pogi** [ProjectNami/projectnami#309]
Throwing **Error establishing a database connection** when i tried to setup wordpress using MSSQL App Id Account. 

How can we set integrated security login settings in wp-config.php?

i want to achieve something like this if its possible "<add name="core" connectionString="Data Source=VD180000.ds.test.mycomp.com;Database=MyWordpressDB;Integrated Security=True;Trusted_Connection=Yes">"

**Comment by patrickebates** [comment]
This might be something which could be enabled.  But what are your thoughts on it being an environment variable rather than a setting in wp-config?  I ask because I see a chicken vs. egg scenario when performing the initial installation if the setting isn't already established.

**Comment by cissemy** [comment]
Hi,
Can I use inhouse sql server instead of azure sql server ?
Thanks

**Comment by patrickebates** [comment]
Yes.  At this time you only need to enable SQL Authentication on your SQL Server and use a SQL account to connect.

Note that you must run at least SQL 2012 (Express is fine).  And I would suggest running the newest version you have available.  We will soon be adding support for new MySQL translations which will need SQL 2017 in order to be enabled.

**Comment by mehmetonurtan34** [comment]
Do you plan to make Integrated Security available in the future?

**Comment by patrickebates** [comment]
Maybe, but aside from this thread there has been no other requests for it.

Do you have any suggestions regarding the installation issue detailed above?  Would the environment variable option be viable for you?
